src/data/zonelist.txt

```diff
@@ -180,7 +180,7 @@ Override	Unsorted	Discoveries
 Events	Events	Events
 
 Twitch	Town	Time-Twitching Tower
-Crimbo23	Holiday	Christmas 2023
+Crimbo24	Holiday	Christmas 2024
```

Comment:
Yeah, fine, whatever. I did this:
```
Crimbo24	Holiday	Christmas 2024
Holiday Islands	Crimbo24	Holiday Islands
```
just in case they put some adventure zones in Crimbo Town itself.

Compare with:
```
Tammy's Offshore Platform	Crimbo19	Crimbo 19 Underwater
```
which has adventure zones in it.